### ISSUE TITLE
problems creating new workspace

### ISSUE THREAD

**Issue by andr222** [vsmejkal/chrome-workspaces#31]
my first disasterous attempt at creating a workspace :
1) I created a workspace, but I was in a window which didn't contain the page that I wanted to include.
The extension created a new window, with an empty tab.
I would have expected it to create an empty workspace.

2) I changed to the window containing the page I wanted to include.

3) I attempted to add the page to the workspace.
The extension transferred the page to the window it had created in step 1.
Replacing the page with an empty page.
I would have expected it to simply add the page to the workspace, without moving it or adding an empty page.

4) I uninstalled the extension and succeeded returning the page to where it was.

Conclusion : 
The lack of documentation and bizarre behavior makes it very dangerous to use this extension.

**Comment by vsmejkal** [comment]
I agree that opening the window after creating a new workspace (step 1) might be misleading. I will try to fix that by navigating back to the list of workspaces instead of opening a new window.

Regarding step 3, I don't exactly follow. How did you attempt to add the page to the workspace?

**Comment by andr222** [comment]
My context:
I need a number of tab groups, to use with my tab manager, which has some functions not available in the navigator.
Some groups have up to a hundred of tabs.  My tab manager requires each group to be in a separate window.

So if I use workspaces in the navigator, there will be only one workspace by window.  But it seems that there will be one empty/unnamed workspace in each window.
Is it possible to avoid this ?
If not, with my current 7 tab groups, I will have 14 lines in the workspace menu for 7 workspaces.

Am I misunderstanding this ?  How can this be corrected ?

**Comment by andr222** [comment]
Sorry for my initial post.  The first time I tried out workspaces.

**Comment by vsmejkal** [comment]
Sorry, but I still don't fully understand the problem. From what you've said there are some duplicates in Workspaces list.

Could you please provide precise steps how to reproduce it? Ideally with a screenshot.

**Comment by andr222** [comment]
In my test, there is an empty entry for the window in addition to the named workspace.
When I questioned that on the internet, someone said it was normal.
My problem, is that in order to work with my tab manager which has special functions not available with Chromium, I need a separate window for each tab group.

Sorry, I don't have a screenshot, since I cancelled my test workspace.
But my  workspace list was  :

(empty line)
named workspace

I tried to put a tab in the empty line, and nothing happened.  Everything in that window was transferred to the workspace when I created the workspace.

I created a second workspace, and could transfer tabs to (and from) that workspace without problem.

I tried creating a workspace in another window, and I got :

(empty line)
workspace1
(empty line)
workspace2

So with 7 workspaces in 7 windows (for my current 7 tab groups), I would have 14 lines in the workspace list.  7 workspaces, and 7 empty lines.

It would be simpler to name the windows in Chromium, but that option doesn't seem to exist.
